py/torch_tensorrt/dynamo/conversion/_TRTInterpreter.py

```diff
@@ -323,6 +324,21 @@ def call_function(self, target: str, args: Any, kwargs: Any) -> Any:
         assert self._cur_node_name is not None
         return converter(self.network, target, args, kwargs, self._cur_node_name)
 
+    def get_attr(self, target: str, args: Any, kwargs: Any) -> np.ndarray:
```

Comment:
So this is to process get_attr nodes and return constants during TRT network building right ? 
- Shouldn't we return constant ITensors ? (or)
- How was this handled before this function ?

Reply:
Yes, that's correct. In many cases, we do not actually want the `ITensor` back, because (for convolution, batch norm, etc.), we can use the values in the Numpy array directly to precompute relevant attributes, and then TRT will cache those for us. If we freeze right away, we lose this utility. There are, however, other cases where we need to freeze because the converter cannot handle non-`ITensor`s. #2306 should address this in a robust way.